040_More_Documentation.html.md

```diff
@@ -13,5 +13,33 @@ This section describes the details of your solution in a logical order. For exam
 
 #### Example
 For an example of unique solution documentation, see the <a href="https://devportal.yaas.io/solutions/saphybrisprofile/index.html">SAP Hybris Profile</a> documentation.
+
+### README files
+
+README files describe other files in a given directory.
+
+When creating external README files, use this template:
+
+````
+```
+Project Title converted to ASCII art using an ASCII generator. For example, http://patorjk.com/software/taag/#p=display&f=Big&t=Project%20Title%0A. Use the following font: Big.
+```
+
+## Overview
+
+A short description of the functionality that the project or chart provides.
+
+## Prerequisites
+
+A list of requirements to run the project or chart.
+
+## Details
+
+Detailed information about the functionality that the project or chart provides. For example, a further description, an installation or configuration guide, operating instructions, examples. Add subsections for better readability.
+
+## Troubleshooting
+
+A list of potential issues and tips on how to avoid or solve those issues.
```

Comment:
List potential issues, and tips on how to avoid or solve these issues.

Need to update this on the SF repo as well.